src/Compilers/CSharp/Portable/Symbols/ConstraintsHelper.cs

```diff
@@ -390,6 +390,62 @@ private static bool CheckConstraintsSingleType(TypeSymbol type, CheckConstraints
         }
 
         public static bool CheckConstraints(
```

Comment:
>public static bool CheckConstraints( [](start = 8, length = 36)

Is anyone checking value returned by this function? Please change to void function otherwise.